### PR TITLE
Enhancement/remove where

### PR DESCRIPTION
**Pull request by AlexandreMaul**
Hi,

I added removeWhere method to remove multiple event from EventController.

### CODE REVIEW

**Comment by AlexandreMaul** [comment]
Hello, any news with this pr ?

File: lib/src/event_controller.dart [hunk 1]
```diff
@@ -78,6 +78,14 @@ class EventController<T> extends ChangeNotifier {
     }
   }
 
+  /// Removes multiple [event] from this controller.
+  void removeWhere(bool Function(CalendarEventData<T> element) test) {
+    for (final e in _events) {
+      e.removeWhere((element) => test(element));
+    }
```
Comment (ParthBaraiya) [inline lib/src/event_controller.dart:85]: This does not remove events from `_rangingEventList`. Please add that as well.

**Comment by PRBaraiya** [comment]
@AlexandreMaul  Please resolve the conflicts in this branch.

**Comment by AlexandreMaul** [comment]
Hello @PRBaraiya, i've commited the fix.

**Review by ParthBaraiya — APPROVED** [review]

**Pull request merged**